doc/user/install.md

```diff
@@ -36,7 +36,7 @@ curl -L https://downloads.mtrlz.dev/materialized-{{< version >}}-x86_64-apple-da
 # Add the signing key for the Materialize apt repository
 apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 379CE192D401AB61
 # Add and update the repository
-apt-add-repository 'deb http://packages.materialize.io/apt/ /'
+echo "deb http://packages.materialize.io/apt/ /" > /etc/apt/sources.list.d/materialize.list
```

Comment:
Well so this has the defect that it requires you to be root, which I think has the opposite problem which is that it is only likely convenient from within Docker? The incantation I see around is this

```
echo "deb http://packages.materialize.io/apt/ /" | sudo tee /etc/apt/sources.list.d/materialize.list
```

but annoyingly sudo isn't present in default Ubuntu Docker... but hopefully a more obvious fix?

Reply:
Spoke offline, we will instead put the `>` under a new shell invocation, so that people can easily prepend it by `sudo` if they get a permissions error.